lib/src/throughput/EwmaEstimator.ts

```diff
@@ -0,0 +1,60 @@
+import type { ResourceTiming } from '../request/ResourceTiming'
+import { Ewma } from './Ewma';
+import type { ThroughputEstimator } from './ThroughputEstimator'
+
+type EwmaEstimatorOptions = {
+	fastHalfLife: number;
+	slowHalfLife: number;
+}
```

Comment:
This needs to be exported in a separate file

Reply:
Addressed it in https://github.com/streaming-video-technology-alliance/common-media-library/pull/160/commits/e3999d78eba5bbb1fee1dd301a086dc46972585f